custom_components/localtuya/light.py

```diff
@@ -63,10 +87,8 @@ def brightness(self):
     @property
     def color_temp(self):
         """Return the color_temp of the light."""
-        try:
+        if self.has_config(CONF_COLOR_TEMP):
             return int(MAX_MIRED - (((MAX_MIRED - MIN_MIRED) / 255) * self._color_temp))
-        except TypeError:
```

Comment:
What happens here if CONF_COLOR_TEMP is not configured? Is there a default value to return?

Reply:
The default value is to return `None`, but I should probably add an explicit return for it.